gebrueder-weiss-woocommerce.php

```diff
@@ -41,4 +43,23 @@ function GbWeiss()
     };
 
     // Initialize the plugin here.
+    $plugin = GbWeiss();
+
+    /**
+     * Validate client credentials
+     * this will later be shifted to the corresponding save / validation workflows
+     */
+    $authenticationClient = new OAuthAuthenticator(new GuzzleHttp\Client());
+    $authenticationClient->setAuthenticationEndpoint('http://18019fdce8ff:8887/token');
```

Comment:
I think we should read this value from the environment variables. The default value should contain the production api endpoint.